### PR TITLE
make krr_scan action available for mattermost sink

### PR DESCRIPTION
**Pull request by IdeoG**
https://robustacommunity.slack.com/archives/C02R0LVANKY/p1712750565337999

### CODE REVIEW

File: src/robusta/integrations/mattermost/sender.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
     KubernetesDiffBlock,
     ListBlock,
     MarkdownBlock,
+    ScanReportBlock,
     TableBlock,
 )
 from robusta.core.reporting.utils import add_pngs_for_all_svgs
```
Comment (arikalon1) [inline src/robusta/integrations/mattermost/sender.py:17]: I think there's a missing import here
`from robusta.core.reporting.consts import EnrichmentAnnotation`

Can you please add it?

File: src/robusta/integrations/mattermost/sender.py [hunk 2]
```diff
@@ -144,7 +147,10 @@ def send_finding_to_mattermost(self, finding: Finding, platform_enabled: bool):
             blocks.append(MarkdownBlock(finding.description))
 
         for enrichment in finding.enrichments:
-            blocks.extend(enrichment.blocks)
+            if enrichment.annotations.get(EnrichmentAnnotation.SCAN, False):
```
Comment (arikalon1) [inline src/robusta/integrations/mattermost/sender.py:150]: I think here it should be:
```
            if enrichment.annotations.get(EnrichmentAnnotation.SCAN, False):
                enrichment.blocks = [Transformer.scanReportBlock_to_fileblock(b) for b in enrichment.blocks]
            
            blocks.extend(enrichment.blocks)
```

The `blocks` array is the collection sent to mattermost later on, so we want to add the converted file to it

**Review by arikalon1 — COMMENTED** [review]
Thanks for the PR @IdeoG 
Left small comments

I don't have a mattermost account, so I did not test it

File: src/robusta/integrations/mattermost/sender.py (hunk 2 above)
Comment (IdeoG) [reply src/robusta/integrations/mattermost/sender.py:151]: You're right. Got it!

**Review by arikalon1 — APPROVED** [review]
Looks good
Thanks for the contribution @IdeoG

**Pull request merged**